decisions/wc_offmap_decisions.txt

```diff
@@ -46,19 +40,21 @@ offmap_decisions = {
 		}
 		
 		third_party_potential = {
-			show_only_failed_conditions = yes
 			FROMFROM = {
 				show_scope_change = no
+
 				prisoner = yes
+				can_be_sacrificed_trigger = yes
+				is_untouchable_trigger = no
 				host = { character = ROOT_FROM }
 			}
 		}
-		
+
 		third_party_allow = {
 			#show_only_failed_conditions = yes
 		}
 		
-		third_party_score = {
+		third_party_score = { # DO NOT FORGET TO CHANGE "caltulate_PREV_offmap_old_gods_currency_insanity_effect" IF YOU CHANGE THIS!
```

Comment:
Please write a comment that it's no longer used in calculations, just shows stuff.

Reply:
Added